packages/runtime/src/interpreter/opcode.ts

```diff
@@ -106,6 +106,28 @@ export class Op {
     return a;
   }
 
+  /**
+   * asserts if given index lies in 64 bit unsigned integer
+   * @param index Index
+   * @param line line number in TEAL file
+   */
+  assert64BitIndex (index: bigint, line: number): void {
+    if (index > 63n) {
```

Comment:
maybe we should add a constant `MAX_UINT6`?

Reply:
yes